### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Completed home page

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

**Review by Kavita539 — APPROVED** [review]

**Comment by Kavita539** [comment]
UI looks good. Great work!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+          <p class="hero-sub-title">Upto <span class="price-off-bg">60% off</span> on all the products</p>
+          <a class="shop-now-link" href="./productList.html"><button class="hero-btn shop-now-btn">Shop Now</button></a>
+        </div>
+      </div>
+
+      <div class="prominent-category">
+        <p class="category-header secondary-color">Prominent Category</p>
+        <div class="category-container">
+          <div class="category-unit flex-center">
+            <img src="./assets/card_camera.jpg" alt="">
```
Comment (ShreyashMahajan) [inline index.html:110]: always try to give name to alt tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+          <div class="category-unit flex-center">
+            <img src="./assets/card_img6.jpg" alt="">
+            <p class="category-content">Laptop</p>
+          </div>
+          <div class="category-unit flex-center">
+            <img src="./assets/watch.jpg" alt="">
+            <p class="category-content">Watch</p>
+          </div>
+          <div class="category-unit flex-center">
+            <img src="./assets/tablet.jpg" alt="">
```
Comment (ShreyashMahajan) [inline index.html:122]: alt = 'tablet image'

File: login.html [hunk 3]
```diff
@@ -25,15 +25,17 @@
...
           >
           <span class="material-icons icon hide-menu"> menu </span>
+          <a class="primary-color" href="./index.html">
             <h2 class="main-title">
               magni<span class="bold-main-title">Z</span>ent
             </h2>
+          </a>
           </div>
 
           <div class="header-main-content">
```
Comment (ShreyashMahajan) [inline login.html:35]: you can use header tag here for semantic html

File: productList.html [hunk 4]
```diff
@@ -16,55 +16,70 @@
...
-                    <div class="icon-unit flex-column flex-center">
-                        <span class="material-icons icon"> favorite </span>
-                        <p>WishList</p>
-                    </div>
-                    <div class="icon-unit flex-column flex-center">
-                        <span class="material-icons icon"> shopping_cart </span>
-                        <p>Cart</p>
-                    </div>
-                </div>
+      <div class="header-container">
```
Comment (ShreyashMahajan) [inline productList.html:19]: header tag can be used here

**Review by ShreyashMahajan — APPROVED** [review]
Look at the comments, Then good to merge. UI is really Awesome

**Pull request merged**